### DISCUSSION TITLE
Anyone willing to work with me to get this supported for GovCloud?

### DISCUSSION THREAD

**Discussion by ikspozd** [BishopFox/cloudfox#11] [Q&A]
Hello Community,

If you're interested in working with me to get this supported for GovCloud, please let me know here.

**Comment by sethsec-bf** [comment]
Hey @ikspozd - This is on our road map, but if you want to take a crack at it, or if you want to brainstorm here, let me know.

**Reply by ikspozd** [reply]
Sorry Seth had other priorities but just tested version 1.10.0-prerelase on GovCloud and it seems to be working now. Thanks!!
